data/p/developer-experience-survey-template.mdx

```diff
@@ -13,6 +13,20 @@ This article aims to help companies who want to survey their engineers to improv
 
 To make a summary of all the important aspects to take into account according to the SPACE framework, you can also find and download our _Engineering productivity cheatsheet_ below:
 
+## DX by Axolo
+
+Axolo recently released a new Slack bot to enable tech teams to measure and improve their developer experience. It's a great tool to help you improve your engineering processes and to measure the impact of your changes. You can find more information about it on [Dx by Axolo](https://axolo.co/developer-experience-survey).
```

Comment:
c'est cool, mais j'ai encore l'impression que le focus est trop sur la solution, pas assez sur le problème. 

Je serai parti sur "After working with engineering teams for 3 years, we've understood that continuous improvements of the working environment of developers is a big issue for engineering managers. It's hard to receive honest feedback, and to know where to focus on to create the best developer experience. 

To answer theses issues we've developed a (100% free forever) survey tool with the survey you're actually reading. Join hundreds of engineering teams that are improving their developer experience with DX by Axolo!"

Reply:
Merci je vais changer j’étais pas mega inspirer  [](edited-with-Axolo)